apps/emqx_auth_mnesia/src/emqx_auth_mnesia.erl

```diff
@@ -72,7 +72,8 @@ check(ClientInfo = #{ clientid := Clientid
         List ->
             case match_password(NPassword, HashType, List)  of
                 false ->
-                    ?LOG(error, "[Mnesia] Auth from mnesia failed: ~p", [ClientInfo]),
+                    Info = maps:without([password], ClientInfo),
+                    ?LOG(info, "[Mnesia] Auth from mnesia failed: ~p", [Info]),
```

Comment:
is the change from `error` to `info` intended ?

Reply:
People complain that it generates tons of messages by bots brutforce.